.github/workflows/job_build_windows.yml

```diff
@@ -155,10 +155,10 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ runner.arch }}-ccache
 
-      - name: Configure Developer Command Prompt for Microsoft Visual C++ (2019)
+      - name: Configure Developer Command Prompt for Microsoft Visual C++
         uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
         with:
-          toolset: 14.29
+          toolset: 14.40 # v2022
```

Comment:
you need to ensure that places where OV provider is used, also use VS 2022, otherwise can have issues like in GenAI when Pybind11 code cannot recognize ov::Tensor compiled with different compiler version.

Reply:
You are right - https://github.com/openvinotoolkit/openvino.genai/pull/1598